### PR TITLE
Add WaitForAllPodsInNamespaceRunning()

### PR DESCRIPTION
**Pull request by anna-savina**
Added WaitForAllPodsInNamespaceRunning()

### CODE REVIEW

**Comment by prabinovRedhat** [comment]
lgtm

File: pkg/pod/list.go [hunk 1]
```diff
@@ -70,3 +71,25 @@ func ListInAllNamespaces(apiClient *clients.Settings, options v1.ListOptions) ([
 
 	return podObjects, nil
 }
+
+// WaitForAllPodsInNamespaceRunning check that all pods in namespace that match options are in running state
+func WaitForAllPodsInNamespaceRunning(
+	apiClient *clients.Settings,
+	nsname string,
```
Comment (kononovn) [inline pkg/pod/list.go:78]: Please validate is nsname is not empty.


	if nsname == "" {
		glog.V(100).Infof('nsname' parameter can not be empty")
		return nil, fmt.Errorf("failed to list pods, 'nsname' parameter is empty")
	}

File: pkg/pod/list.go [hunk 2]
```diff
@@ -70,3 +71,25 @@ func ListInAllNamespaces(apiClient *clients.Settings, options v1.ListOptions) ([
...
+	options v1.ListOptions,
+	timeout time.Duration,
+) (bool, error) {
+	podList, err := List(apiClient, nsname, options)
```
Comment (kononovn) [inline pkg/pod/list.go:82]: Please add informative glog message at the beginning of this function

**Review by kononovn — CHANGES_REQUESTED** [review]
This function looks good. Please fix 2 small comments. Thanks

**Review by kononovn — COMMENTED** [review]
lgtm

**Pull request merged**